browser/src/UI/components/ToolTip.tsx

```diff
@@ -17,10 +17,11 @@ export interface IToolTipsViewProps {
 }
 
 export class ToolTipsView extends React.PureComponent<IToolTipsViewProps, {}> {
-
     public render(): JSX.Element {
         const toolTipElements = this.props.toolTips.map((toolTip) => {
-            return <ToolTipView {...toolTip} borderColor={this.props.borderColor} foregroundColor={this.props.foregroundColor} backgroundColor={this.props.backgroundColor}/>
+            const key = toolTip.options.position.pixelX + toolTip.options.position.pixelY
```

Comment:
Are `pixelX` and `pixelY` numbers? If so this can have ambiguous keys, e.g. X=50, Y=50 | X=100, Y=0 | etc.

Maybe `${pixelX},${pixelY}` would be better.